scripts/monitoring/cron-send-rkhunter-checks.py

```diff
@@ -0,0 +1,77 @@
+#!/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+"""
+ This script is used to check the rkhunter logs for infections and report to Zabbix.
+ It will report an issue to Zabbix if any lines match 'infected' or 'Warning'.
+"""
+
+# Reason: disable pylint import-error because our modules aren't loaded on jenkins.
+# Reason: disable pylint invalid-name because it does not like the naming
+# of the script to be different than the class name
+# pylint: disable=invalid-name,import-error
+
+from __future__ import print_function
+
+import os
+import re
+import yaml
+
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+
+
+class CheckStatus(object):
+    """ Class to check for issues found in rkhunter logs. """
+
+
+    def __init__(self):
+        self.yaml_config = None
+
+
+    @staticmethod
+    def check_rkhunter(log_message, logfile):
+        """ Check number of occurrences of the provided string in the specified logfile. """
+
+        total_issues = 0
+
+        if os.path.isfile(logfile):
+            with open(logfile) as open_file:
+                stripped_line = list([line.rstrip() for line in open_file.readlines()])
+                for line in stripped_line:
```

Comment:
Looks like a double loop in your code here.  First loop walks the lines and strips and then directly after you proceed to loop again.  Is this the case?  I'd prefer to walk the loop once if we can.